js/languages/en-US.json

```diff
@@ -701,7 +701,8 @@
       "sendPaymentFailDialogTitle": "Unable to send payment",
       "sendConfirm": {
         "heading": "Are you sure?",
-        "body": "Double check that the address and amount are correct.",
+        "body": "A %{fiatBtcPairing} estimated network fee will be included",
```

Comment:
Maybe change this to "A network fee will be included, currently estimated at %{fiatBtcPairing}"

That way it's more clear it can change by the time you click send, and it makes it sound more like a guess (which it is).